### ISSUE TITLE
Easy to Read in Accessibility of machine learning and generative AI

### ISSUE THREAD

**Issue by LuisMalhadas** [w3c/process#1035]
The document speaks of plain language but not **easy to read** text.
Easy to read text follows stricter normative, here is an example: https://www.inclusion-europe.eu/wp-content/uploads/2017/06/EN_Information_for_all.pdf

Here I share a csv of a set of documents over the subject (including some papers about generative AI applied to easy2read):

[Accessibles-Easy2Read.csv](https://github.com/user-attachments/files/19889994/Accessibles-Easy2Read.csv)

The statement: 
```text
For example, a translation of the popular poem, “Mary Had a Little Lamb”, into a non-English language, and then back to English observes the error of “it’s fleece was white as snow” being converted to “it snowed sheep hair”. 
```
Does not mention the models used nor the source.
While it may be true, for the mentioned instance, it is stated as a linear deterministic event, which is not.
Generative AI is non-deterministic, and therefore one can't state one single example as if it always follows that path. Although it may struggle with translating meanings, it is only due to lack of context, and lack of proper curated training data.
You can have better results given better data.

The document also lacks, in general, a consideration towards the cognitive accessibility.
Another example of cognitive accessibility that generative AI will enhance, is the generation of **pictograms**, by allowing a non-deterministic generation of pictograms that can iteratively improve the process of communicating meaning and semantics.

**Comment by frivoal** [comment]
I am unsure what specifically you're proposing here. Everyone agrees that making the document easy to read is important, but given the nature of this document, precision is important too. If you have specific edits you would like to propose, I suggest you open individual issues about them.

As for usage of Generative AI, I think it would be broadly unacceptable to let such a system just rewrite / enhance the document on its own. How you want to derive insights into what should be adjusted is up to you, and I don't think there's anything that says you cannot use Generative AI for doing that, but do note that you need to abide by the https://www.w3.org/community/about/process/cla/ in order to make contributions to this group, which expects contributors to have control over the copyright of material they submit. If you're not yourself the author of your contributions, I would encourage you to read that text carefully to make sure you can comply with it. (I am not a lawyer, this is not legal advice).

With that in mind, I would suggest closing this issue (and opening specific ones as necessary). Does that seem reasonable?

**Comment by LuisMalhadas** [comment]
First of all, thank you for the reply.
Being new here, I tried to follow the rules for contributing, but for sure will fail to do so.

Either way, I was proposing to add a new use cases, similar to the one of the plain language, that follows a less restrictive standard, about easy to read text that follows a more restrictive standard, as well as about Pictograms. In both cases Generative AI can be used to enhance the process of understanding content.

Besides that, I was wondering or trying to propose an edit about the Plain Language provided example, to mention the models used, and to be more clear about its non-determinism, or making clear that just a sentence is not so much context.

I can open new issues about those proposals. 3 new issues?

In any way was I suggesting to use AI to change this document, if that is what you mean when you say:

>  I think it would be broadly unacceptable to let such a system just rewrite / enhance the document on its own.

Also, not being a lawyer, I don't exactly understand what you mean by being the author of the contribution, as all I have written, so far, are my own words... Although, I do usually cite others, I mean, I did not invent nor did I claim any authorship over Easy to Read standard or Pictograms. Should the people who wrote the standard be the ones coming here and write the contribution? Is that it?

Here is the thing, I am developing a system that is complementing (not replacing) online content with the generation of Easy to Read, and Pictograms that are making such content easier to understand. We have the cooperation of many human validators, as per the rule of the standard, and the progress we have made show very promising results. So when I saw the document about W3C group including/discussing Generative AI to help in accessibility I thought that what we are doing is missing from the examples/use cases.

Should I go on with the suggestions, and open new individual issues?

**Comment by LuisMalhadas** [comment]
I am talking about this document:

https://raw.githack.com/w3c/ai-accessibility/FPWD-ai-accessbility/index.html#sign-language

In it there is a link that led me here.

**Comment by frivoal** [comment]
Ah, this is the wrong place to talk about that document, and the confusion you got in response to your suggestion is because this group is not the maintainers of the document you are referring to. You want to direct your feedback to https://github.com/w3c/ai-accessibility/

**Comment by LuisMalhadas** [comment]
Thank you. And sorry for the error.
Closing the issue.

**Comment by LuisMalhadas** [comment]
I missed that.
Thank you.

And sorry again, for all the confusion created.
